packages/server/src/integration-test/postgres.spec.ts

```diff
@@ -27,7 +27,9 @@ describe("row api - postgres", () => {
   let makeRequest: MakeRequestResponse,
     postgresDatasource: Datasource,
     primaryPostgresTable: Table,
-    auxPostgresTable: Table
+    o2mInfo: ForeignTableInfo,
```

Comment:
NAB: `o2m` is a bit odd. Personally I'd prefer `oneToManyInfo`

Same applied to `m2m`, etc

Reply:
Would agree - more verbose variable names are preferred.

Reply:
Always the fight between verbosity and not ultra-long variables. But I'll change it :)